koku/api/report/ocp/provider_map.py

```diff
@@ -157,7 +158,10 @@ def __init__(self, provider, report_type):
                         "custom": ProviderAccessor(Provider.PROVIDER_OCP).infrastructure_key_list,
                     },
                 },
-                "group_by_options": ["cluster", "project", "node"],
+                "group_by_options": ["cluster", "project", "node", "persistentvolumeclaim"],
+                # clean_group_lable converts the db field to a preferred label for the api return
+                # see function _clean_grouping_lables
+                "clean_group_label": {"persistentvolumeclaim": "pvc", "persistentvolumeclaims": "pvcs"},
```

Comment:
```suggestion
                "clean_group_label": {"persistentvolumeclaim": "pvc", "persistentvolumeclaims": "pvcs"},
```
Why do we have `pvc` and `pvcs` here? When is one or the other used?

Reply:
We automatically add an `s` to our group by options. So, if we group by `pvc` we will get the expected `pvcs` in the return.